CMakeLists.txt

```diff
@@ -29,6 +29,7 @@ option(USE_MKL "Compile with MKL support" ON)
 option(USE_MPI "Use MPI library" OFF)
 option(USE_NCCL "Use NCCL library" ON)
 option(USE_SENTENCEPIECE "Download and compile SentencePiece" ON)
+option(USE_TCMALLOC "Use TCMALLOC if available" ON)
```

Comment:
Is this commit required for python bindings? Or is it just a "while I'm here" kind of commit?